src/components/admin/SpaceCard.tsx

```diff
@@ -132,7 +132,11 @@ export const SpaceCard: React.FC<SpaceCardProps> = ({
           </p>
         </div>
         <div className="flex justify-center align-center mt-4">
-          <LinkButton href={editSpaceUrl} disabled={!isEditable} width="6/12">
+          <LinkButton
+            href={editSpaceUrl}
+            disabled={!isEditable}
+            width="halfContainer"
```

Comment:
[NOTE] Few musings:
Is this an exception? Would normally buttons be full width and only rarely half width? 

On that note, the strings can be `full` and `half`. Easy to deal with the `.tailwind.ts` names